ddl/ddl.go

```diff
@@ -582,6 +582,7 @@ func (d *ddl) doDDLJob(ctx sessionctx.Context, job *model.Job) error {
 		return errors.Trace(err)
 	}
 	ctx.GetSessionVars().StmtCtx.IsDDLJobInQueue = true
+	defer func() { ctx.GetSessionVars().StmtCtx.IsDDLJobInQueue = false }()
```

Comment:
IsDDLJobInQueue is used to mark whether the DDL job is put into the queue. 
This originally refers to whether the DDL job has been placed in the queue, and the description may not be very clear. This variable was originally used to handle scenarios that encountered this [problem](https://github.com/pingcap/tidb/pull/6797) before being placed on the queue.

Reply:
ok. I will recover this logic.